### PR TITLE
Added RCON support

### PR DESCRIPTION
**Pull request by derNiklaas**
This pull requests adds support for rcon clients.

npm package used: rcon-client
https://www.npmjs.com/package/rcon-client
https://github.com/janispritzkau/rcon-client

### CODE REVIEW

File: nodecg-io-rcon/extension/index.ts [hunk 1]
```diff
@@ -0,0 +1,80 @@
+import { NodeCG } from "nodecg/types/server";
+import { NodeCGIOCore } from "nodecg-io-core/extension";
+import { Service, ServiceProvider } from "nodecg-io-core/extension/types";
+import { emptySuccess, success, error, Result } from "nodecg-io-core/extension/utils/result";
+import { Rcon } from "rcon-client";
+import { config } from 'process';
```
Comment (daniel0611) [inline nodecg-io-rcon/extension/index.ts:6]: This import isn't used, so it can be safely removed.
```suggestion
```

File: package.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
     "name": "nodecg-io",
     "private": true,
     "scripts": {
-        "postinstall": "cd nodecg-io-core && npm link && cd dashboard && npm link nodecg-io-core && npm i && cd ../../nodecg-io-twitch && npm link nodecg-io-core && npm link && cd ../sample && npm link nodecg-io-twitch nodecg-io-core && npm i",
+        "postinstall": "cd nodecg-io-core && npm link && cd dashboard && npm link nodecg-io-core && npm i && cd ../../nodecg-io-twitch && npm link nodecg-io-core && npm link && cd ../nodecg-io-rcon && npm link && cd ../sample && npm link nodecg-io-twitch nodecg-io-core nodecg-io-rcon && npm i",
```
Comment (daniel0611) [inline package.json:5]: Small error here. We will get rid of this mess with #7 soon anyway.
```suggestion
        "postinstall": "cd nodecg-io-core && npm link && cd dashboard && npm link nodecg-io-core && npm i && cd ../../nodecg-io-twitch && npm link nodecg-io-core && npm link && cd ../nodecg-io-rcon && npm link nodecg-io-core && npm link && cd ../sample && npm link nodecg-io-twitch nodecg-io-core nodecg-io-rcon && npm i",
```

File: nodecg-io-rcon/extension/index.ts (hunk 1 above)
Comment (derNiklaas) [reply nodecg-io-rcon/extension/index.ts]: Implemented in https://github.com/codeoverflow-org/nodecg-io/pull/16/commits/71b44abec69963272d4dfb92558ca096d059dd9d

File: package.json (hunk 2 above)
Comment (derNiklaas) [reply package.json]: Changed in https://github.com/codeoverflow-org/nodecg-io/pull/16/commits/71b44abec69963272d4dfb92558ca096d059dd9d

**Pull request merged**